types-bundle/src/types.ts

```diff
@@ -328,6 +327,7 @@ export const TYPES_900_undefined_deprecated: RegistryTypes = {
 };
 
 export const TYPES_POST_900: RegistryTypes = {
+  ...TYPES_800_899,
```

Comment:
This is confusing, form the name i would expect only types post runtime 900.

Reply:
agreed it is confusing, which is why it was broken previously.